### PR TITLE
dynamic busy box repo

### PR DESCRIPTION
**Pull request by cncoble**
Make busybox repo reference dynamic

### CODE REVIEW

File: stable/dynamic-gateway-service/values.yaml [hunk 1]
```diff
@@ -12,6 +12,7 @@ datapower:
     pullPolicy: IfNotPresent
     pullSecrets: 
     - name: datapower-pull-secrets
+    busyboxrepo: busybox
```
Comment (kraman) [inline stable/dynamic-gateway-service/values.yaml]: This should be its own image section...
```
datapower:
  image:
    repository: ibmcom/datapower
    tag: 7.7.1.1.300826
    pullPolicy: IfNotPresent
  busybox:
    repository: busybox
    tag: 1.29-glibc
    pullPolicy: IfNotPresent
imagePullSecrets:
  - name: datapower-pull-secrets
```

Comment (cncoble) [reply stable/dynamic-gateway-service/values.yaml]: ok will update thank you

**Comment by cncoble** [comment]
@kraman updated

File: stable/dynamic-gateway-service/templates/statefulset.yaml [hunk 2]
```diff
@@ -39,7 +39,7 @@ spec:
       {{- end }}
       initContainers:
         - name: dpinit
-          image: busybox
+          image: {{ .Values.datapower.busybox.repository | default "busybox" }}
```
Comment (kraman) [inline stable/dynamic-gateway-service/templates/statefulset.yaml]: @cncoble I think you forgot to update to use the tag and pull policy

**Comment by cncoble** [comment]
Added conditional logic to use rep, tag, and pull policy when defined in Values  otherwise fall back to previous behavior

**Comment by cncoble** [comment]
@kraman ^

**Pull request merged**